doc/examples/segmentation/plot_regionprops_table.py

```diff
@@ -0,0 +1,106 @@
+"""
+===================================================
+Explore and visualize region properties with pandas
+===================================================
+
+This toy example shows how to compute the size of every labelled region in a
+series of 10 images. We use 2D images and then 3D images. The blob-like
+regions are generated synthetically. As the volume fraction (i.e., number of
+pixels or voxels covered by the blobs) increases, the number of blobs
+(regions) decreases, and the size (area or volume) of a single region can get
+larger and larger. The area (size) values are available in a pandas-compatible
+format, which makes for convenient data analysis and visualization.
```

Comment:
```suggestion
format, which makes subsequent data analysis and visualization convenient.
```

Reply:
I don't find the suggested change better than the original...

Reply:
If it is a correct English, I am more than happy to keep your version :).

Reply:
Here's a reference for you: https://www.collinsdictionary.com/dictionary/english/make-for ;)